script/Arcueid/Day 10/10_00_ARC10_1.txt

```diff
@@ -31,7 +31,7 @@ As I bask in the morning's warm welcome―――
 [sha:6cbbd7b766f75ada376a8dfb0bfea9c54ddc9de8]{
 -- Page 2, Offset 24265.
 -- 無意識のうちに枕元の眼鏡を手に取る。
-I put on the glasses resting by my pillow,
+I put on the glasses resting by my nightstand,
```

Comment:
on my nightstand

Reply:
completely missed this comment yesterday sorry :aichanteehee: